server/pbench/bin/state/config/pbench-server.cfg

```diff
@@ -1,151 +1,53 @@
-
-[pbench-files]
-pbench-top-dir = %(deploy-pbench-dir)s
-pbench-backup-dir = %(deploy-pbench-backup-dir)s
-pbench-logs-dir = %(deploy-pbench-logs-dir)s
-pbench-tmp-dir = %(deploy-pbench-tmp-dir)s
-
 ###########################################################################
 ## runtime section
 ###########################################################################
 [sosreport]
 user = %(default-user)s
-# CHANGE ME!
 host = pbench-sosreports.example.com
 dir = /path/to/sosreport/dir
-# CHANGE ME!
-mailto = admins@example.com
 
 ###########################################################################
-## deployment section
+## Deployment section
 ###########################################################################
 [pbench-server]
-user = %(default-user)s
-group = %(default-group)s
-mailfrom = %(default-user)s@%(default-host)s
-install-dir = %(default-deploy-dir)s
-script-dir = %(deploy-script-dir)s
-archive-dir = %(deploy-archive-dir)s
-quarantine-dir = %(deploy-pbench-local-dir)s/quarantine
-receive-dir-001 = %(deploy-pbench-receive-dir-001)s
-receive-dir-002 = %(deploy-pbench-receive-dir-002)s
-crontabdir = %(deploy-lib-dir)s/crontab
-
-# CHANGE ME!
-# roles = pbench-results, pbench-backup
-roles = pbench-prep, pbench-results, pbench-backup, pbench-rsync-satellites
-
-# when a tarball comes in to this server, what processing should it undergo?
-# production-like servers do at least the first two and possibly more of these:
+environment = unit-test
+pbench-top-dir = %(unittest-dir)s/pbench
+pbench-local-dir = %(unittest-dir)s/pbench-local
+# Add role for sync'ing with satellites
+roles = pbench-prep, pbench-results, pbench-backup, pbench-sync-satellites
+# Ensure all dispatch states are used
 dispatch-states = TO-UNPACK, TO-INDEX, TO-COPY-SOS, TO-BACKUP
 
-# we need to install some stuff in the apache document root
-# so we either get it directly or look in the config file.
-# N.B. Different distros use different config files.
-# The following works on Fedora, RHEL, CentOS.
 [apache]
-# configfile = /etc/httpd/conf/httpd.conf
 documentroot = %(unittest-dir)s/var-www-html
 
-# this *has* to agree with the setting in the pbench-agent config file
-[results]
-
-# CHANGE ME!
-webserver = pbench.example.com
-host_info_url = http://%(webserver)s/pbench-results-host-info.versioned/pbench-results-host-info.URL
-
 ###########################################################################
 # crontab roles
 [pbench-results]
-host = %(default-host)s
-# CHANGE ME!
-tasks = pbench-unpack-tarballs, pbench-copy-sosreports, pbench-edit-prefixes, pbench-index
+# Expand the results role's tasks to add pbench-move-unpacked
+tasks = pbench-dispatch, pbench-unpack-tarballs, pbench-move-unpacked, pbench-copy-sosreports, pbench-edit-prefixes, pbench-index
 
-[pbench-backup]
+# The definition of the crontab role for sync'ing satellite pbench servers.
+[pbench-sync-satellites]
 host = %(default-host)s
```

Comment:
Remove `host = ...`

Reply:
Removed, should be available in the next patch.

Reply:
Well, no, we actually need this here like the other default roles that have it.  No change being made.